packages/pass-style/src/types.d.ts

```diff
@@ -22,7 +22,7 @@ export type PrimitiveStyle =
   | 'string'
   | 'symbol';
 
-export type ContainerStyle = 'copyRecord' | 'copyArray' | 'tagged';
+export type ContainerStyle = 'copyRecord' | 'copyArray' | 'tagged' | 'selector';
```

Comment:
I don’t believe selector qualifies as a container. It’s more like a string. But, the reïfication of a selector is an object, so that might be the distinguishing feature for containers. I do not know.

Reply:
its a container for a string! yeah ok was mostly just following 'tagged' around the code base

Reply:
It's definitely not a container. Our best documentation is at https://endojs.github.io/endo/types/_endo_pass_style.Passable.html , and we could describe Selector as primitive if we're taking a data model perspective, or otherwise a new special case (because, unlike the existing primitives, `Object.is` will differentiate identical values).